addons/auth_signup/views/auth_signup_login_templates.xml

```diff
@@ -103,4 +103,23 @@
 
             </t>
         </template>
+
+    <template id="auth_signup.login_successful" name="Login successful">
+        <t t-call="web.login_layout">
+            <div class="oe_login_form">
+                <p class="alert alert-danger" t-if="error" role="alert">
```

Comment:
do you really need this warning ? (couldn't find a way to have an error on this page, maybe I missed something)